sre/src/sre/plan/op.clj

```diff
@@ -18,7 +18,7 @@
 (extend-type Op
   IBind
   (bind-map [this var-map]
-    (let [replace (partial fmap (fn [v] (fmap (fn [p] (fmap #(var-map %1) p)) v)))]
+    (let [replace (partial fmap (fn [v] (println v)(fmap (fn [p] (println "p" p)(fmap #(do (println %) (var-map %)) p)) v)))]
```

Comment:
why println?

Reply:
sheet, i left it while debugging. nice catch!